_pages/governance.md

```diff
@@ -150,22 +152,22 @@ All participants in the community are encouraged to provide support for new user
 
 Anyone, capable of showing respect to others, can contribute to the project, regardless of their skills, as there are many ways to contribute. For instance, a contributor might be active on the project mailing list and issue tracker, or might supply patches. The various ways of contributing are described in more detail in a separate document [Submitting Contributions](/contributing).
 
-The developer mailing list is the most appropriate place for a contributor to ask for help when making their first contribution. See the [Submitting Contributions](/contributing) page on the gem5 wiki for details of the gem5 contribution process. Each new contribution should be submitted as a patch to our Review Board site. Then, other gem5 developers will review your patch, possibly asking for minor changes. After the patch has received consensus (see [Decision Making Process](#decision-making-process)), the patch is ready to be committed to the gem5 tree. For committers, this is as simple as pushing the changeset. For contributors, a committer should push the changeset for you. If a committer does not push the changeset within a reasonable window (a couple of days), send a friendly reminder email to the gem5-dev list. Before a patch is committed to gem5, it must receive at least 2 “Ship its” from reviewboard. If there are no reviews on a patch, users should send follow up emails to the gem5-dev list asking for reviews.
+[GitHub discsussions](https://github.com/orgs/gem5/discussions) or [GitHub issues](https://github.com/gem5/gem5/issues) are the most appropriate place for a contributor to ask for help when making their first contribution. See the [Submitting Contributions](/contributing) page on the gem5 wiki for details of the gem5 contribution process. Each new contribution should be submitted as a patch to our GitHub site. Then, other gem5 developers will review your patch, possibly asking for minor changes. After the patch has received consensus (see [Decision Making Process](#decision-making-process)), the patch is ready to be committed to the gem5 tree. For maintainers, this is as simple as pushing the changeset. For contributors, a maintainer should approve and merge the changeset for you. If a maintainer does not merge the changeset within a reasonable window (a couple of days), send a friendly reminder email to the gem5-dev list. Before a patch is committed to gem5, it must receive at least 2 approvals. If there are no reviews on a patch, users should send follow up notes asking for reviews.
```

Comment:
"asking for minor changes" seems somewhat unrealistic given how many patches have lengthy back and forths. Maybe just remove "minor"?

Reply:
Also I don't believe at least 2 approvals are required anymore, no?

Reply:
I removed "minor." I also removed "push" since we use github merge now. 

For two approvals, I don't want to change that while making clerical updates. We haven't officially agreed to change that policy, though in practice it's ignored much of the time.